src/pages/RepoPage/RepoPage.spec.tsx

```diff
@@ -245,6 +297,12 @@ describe('RepoPage', () => {
             })
           )
         )
+      }),
+      graphql.query('DetailOwner', (req, res, ctx) => {
+        return res(ctx.data({ owner: mockOwner }))
```

Comment:
do we need to mock the status here? Curious how this is working otherwise

Reply:
I can't find the docs for the version we're on anymore, however iirc it's implicitly set to 200. I did just copy this over from another test.

Reply:
I can add it in for consistency tho